[DOCS-7526] adding support for US5 and EU - updating banners (#22362)
DataDog/documentation: content/en/mobile_app_testing/mobile_app_tests/_index.md

  text: "Best practices for maintaining end-to-end tests"
---

{{< site-region region="us,us5,eu" >}}

> <div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

{{< /site-region >}}

{{< site-region region="us3,ap1" >}}

> <div class="alert alert-warning">Support for Mobile Application Testing is coming soon.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```
